lib/email/receiver.rb

```diff
@@ -914,7 +896,13 @@ def embedded_email_raw
     end
 
     def embedded_email
-      @embedded_email ||= embedded_email_raw.present? ? Mail.new(embedded_email_raw) : nil
+      @embedded_email ||= if embedded_email_raw.present?
+        mail = Mail.new(embedded_email_raw)
+        Email::Validator.ensure_valid_address_lists(mail)
```

Comment:
Didn't you add the `!` at the end of the method's name?

Reply:
I don't know how I managed to stage only some changes. 😅